packages/cron-job/Event/PreCronJobExecutionEvent.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Draw\Component\CronJob\Event;
+
+use Draw\Component\CronJob\Entity\CronJobExecution;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class PreCronJobExecutionEvent extends Event
+{
+    public function __construct(
+        private CronJobExecution $execution,
+        private bool $executionCancelled = false,
```

Comment:
add a property command set from the CronJobExecution and get/setter to change the value.

The original CronJob should not be affected.